### PR TITLE
Citygram::Routes::Page is loaded if ENV ROOT_CITY_TAG

### PR DESCRIPTION
**Pull request by dirkkelly**
[https://citygram-nyc-codeforamerica.herokuapp.com/](https://citygram-nyc-codeforamerica.herokuapp.com/)

[screen shot 2015-04-13 at 11 06 49 pm](https://citygram-nyc-codeforamerica.herokuapp.com/)

### CODE REVIEW

**Comment by dirkkelly** [comment]
This is the minimum changes we need at BetaNYC to support using the same codebase as codeforamerica.

If the environment variable `ROOT_CITY_TAG` is set, this city is loaded instead of the index.

**Comment by dirkkelly** [comment]
@bigfleet are you able to give me feedback on this?

File: spec/routes/page_spec.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        get '/'
+        expect(last_response.body).not_to be_empty
+      end
+    end
+
+    context 'ROOT_CITY_TAG is not set' do
+      it 'raises an exception' do
+        expect do
+          get '/'
+        end.to raise_error Citygram::Models::City::NotFound
```
Comment (bigfleet) [inline spec/routes/page_spec.rb]: I don't think I want this raising an exception when ROOT_CITY_TAG is not set?  What am I missing?

This still has to serve our homepage when it's not sent.

Comment (dirkkelly) [inline spec/routes/page_spec.rb]: Agreed, this still works for CodeForAmerica's instance as this class is loaded conditionally based on the presence of the environment variable.


https://github.com/codeforamerica/citygram/pull/182/files#diff-0a400c8217d53ff4f978163d7c61868cR10

That would only be that case if you navigated to the the `Citygram::Routes::Page` route. 

These routes are loaded conditionally based on the environment variable, so this is truly an edge case that I was only covering for completeness sake.

The alternative would be to chat this conditional to not fallback to nil and instead raise the missing environment variable error.

**Comment by bigfleet** [comment]
Awesome, can I get you to add a couple of documentation lines around that section to that effect?  I love the mechanism you selected for making the switch, let's make sure others can identify it.

Also, is there any chance I could get you to consider appending the code that could redirect https://www.citygram.org/new-york to http://www.citygram.nyc/  to this request?  :smile: :ok: :smiley: 

Obviously that could be a different PR.  Let me know your thoughts.  @dirkkelly

**Comment by dirkkelly** [comment]
@bigfleet sorry for the delay, I've added more instructions to the README

**Comment by dirkkelly** [comment]
@bigfleet wondering if this is alright to merge?

**Pull request merged**